src/panels/config/logs/error-log-card.ts

```diff
@@ -74,24 +91,35 @@ class ErrorLogCard extends LitElement {
   private async _refreshErrorLog(): Promise<void> {
     this._errorHTML = this.hass.localize("ui.panel.config.logs.loading_log");
     const log = await fetchErrorLog(this.hass!);
+    this._isLogLoaded = true;
 
     this._errorHTML = log
-      ? log.split("\n").map((entry) => {
-          if (entry.includes("INFO"))
-            return html`<div class="info">${entry}</div>`;
-
-          if (entry.includes("WARNING"))
-            return html`<div class="warning">${entry}</div>`;
-
-          if (
-            entry.includes("ERROR") ||
-            entry.includes("FATAL") ||
-            entry.includes("CRITICAL")
+      ? log
+          .split("\n")
+          .filter(
```

Comment:
This filters per row, meaning you miss valuable trace info on the lines below?

Reply:
I think it's by design, but do you have any other suggestion?
I have a separate branch where I added highlighted filtered keywords there, but it's not ready yet + I don't want this PR to grow overly big.